docs/usage.md

```diff
@@ -612,20 +612,19 @@ git clone -b sample-app https://github.com/cachito-testing/cachi2-generic.git
 ```
 
 #### Pre-fetch dependencies (generic fetcher)
-In order to retrieve the archive with the tool, a `generic_lockfile.yaml` needs to be in the repository. You can find a
-sample lockfile below. It is identical to one found in the [sample repository](https://github.com/cachito-testing/cachi2-generic/tree/sample-app).
+In order to retrieve the archive with the tool, either a `artifacts.lock.yaml` needs to be in the repository, or an absolute
+path needs to be supplied in the JSON input, pointing to a lockfile. You can find a sample lockfile below. It is identical
+to the one found in the [sample repository](https://github.com/cachito-testing/cachi2-generic/tree/sample-app).
 A lockfile for the generic fetcher must contain a `metadata` header and a list of artifacts, where each artifact is
-represented as a pair of URL and a map containing one or more items in the shape of `"hashing algorithm": "checksum"`:
+represented as a pair of URL and a checksum string in the format of `"algorithm:checksum"`:
 
 ```
 ---
 metadata:
   version: "1.0"
 artifacts:
   - download_url: "https://github.com/jeremylong/DependencyCheck/releases/download/v11.1.0/dependency-check-11.1.0-release.zip"
```

Comment:
Should we also add the `filename` attribute here? I noticed this attribute isn't mentioned anywhere in the documentation.

Reply:
Added.